library-best-practices/JLBP-2.md

```diff
@@ -9,12 +9,12 @@
     longer than your dependencies. Historically this was a
     big problem for Guava, which made a major version bump
     every 6 months, although Guava no longer does this.
-- Use package-protected packages/classes when the API is internal
+- Use package-protected classes and methods when the API is internal
   only and should not be used by consumers.
 - Do not mark methods and classes public by default.
   Assume non-public until a need is known.
 - Prefer fewer packages over more packages to avoid
   unnecessarily publicizing internal details,
-  since any dependencies across package boundaries need to be
-  public. (We may be able to revisit this when we can rely on
-  Java 11 or later.) 
+  since any dependency across package boundaries need to be
+  public. (We may revisit this when we can rely on
```

Comment:
Java 11 was correct. We're never going to support 9 or 10 or other non-LTS releases.